### ISSUE TITLE
[Redmine #13371] icinga2 libbase segfault

### ISSUE THREAD

**Issue by icinga-migration** [lazyfrosch/icinga2-test#110]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/13371

    Author: arno
    Assignee: (none)
    Status: Rejected
    Target Version: (none)
    Created: 2016-11-30T13:37:13Z (closed on 2016-12-06T07:29:07Z)
    Last Update: 2016-12-12T14:33:18Z (in Redmine)

    Icinga Version: 2.5.4
    Backport?: Not yet backported
    Include in Changelog: 1

---

Dear Icinga2 devs,

we are getting icinga2 segfault almost every week.

you could possibly link this report with https://dev.icinga.com/issues/12650

we have followed the doc https://docs.icinga.com/icinga2/latest/doc/module/icinga2/toc\#!/icinga2/latest/doc/module/icinga2/chapter/development#development-debug-core-dump in order to enable core dumps.

We have the core dump files from two icinga2 satellite servers in size 90MiB and 72MiB (gzipped)

Please let me know where could I securely upload the core dump files for the analysis. (or send it in private)

Kind regards,
Andrey Arapov

---

**Relations**:

* relates [#12718](https://dev.icinga.com/issues/12718)
* relates [#13151](https://dev.icinga.com/issues/13151)

---

**Updated by arno on 2016-11-30T14:15:49Z**

* File added _icinga2-satellite04-backtrace.txt_

**Update**

Since we are having some sensitive data (passwords in plaintext) in our icinga2 core dump,
I will post here the backtrace at the moment.

Please find icinga2-satellite04-backtrace.txt (the backtrace only) is attached.

**Edit:**

    [Wed Nov 30 10:06:16 2016] icinga2[414]: segfault at 28 ip 00002af76cc55ec2 sp 00002af78dd0ec50 error 4 in libbase.so[2af76cb04000+1e5000]
    -rw-------.  1 icinga icinga 450M Nov 30 10:06 core.icinga2.27689

Please let me know if there any piece of information we can give you in order to help troubleshoot/find the cause of the problem.

Kind regards,
Andrey Arapov

---

**Updated by mfriedrich on 2016-12-05T09:56:01Z**

* **Relates** set to _12718_

---

**Updated by mfriedrich on 2016-12-05T09:56:14Z**

* **Relates** set to _13151_

---

**Updated by gbeutner on 2016-12-06T07:29:07Z**

* **Status** changed from _New_ to _Rejected_

After careful examination I believe that this is a duplicate of [\#12718,](https://dev.icinga.com/issues/12718) i.e. it should already be fixed in the master branch. Can you re-test this with the current snapshots?

---

**Updated by arno on 2016-12-12T14:33:18Z**

Hello,

we are waiting for the release of 2.6.0.
So we can confirm only after 2.6.0 is released.

Is there any approximate date when 2.6.0 is going to be released?

Thanks,

Andrey Arapov
